.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Please add `.vscode/settings.json` to the `.gitignore` so that it is not part of this PR